Resources/Locale/en-US/corvax/station-goal/station-goals.ftl

```diff
@@ -0,0 +1,224 @@
+    2. Engagement: The goal should require the cooperation of as many departments as possible in the plans;
+    3. Scope: The goal should involve sufficient, but not excessive, amounts of work to ensure the effective completion of the task.
+
+    Please note that distribution of the contents of this document to persons who do not have legal immunity is strictly prohibited due to the possibility of discrediting the management of the Corporation.
+    Therefore, in order to present a new goal to the crew, the command staff must contact Central Command for approval of your ideas.
+
+station-goal-anomalies=
+    Dear station Command, the purpose of your shift is to provide new information about anomalies to NanoTrasen.
+
+    It is necessary to conduct experimental studies aimed at testing the consequences of the collapse of at least 4 unique anomalies.
+    During or after the experiments, it is necessary to isolate and document the aforementioned anomalies.
+
+    Document requirements:
+    1. The official name of the anomaly;
+    2. Physical description;
+    3. Passive properties;
+    4. Reaction of the anomaly to different particles;
+    5. Consequences of the collapse;
+    6. Location of the anomaly.
+
+    The document must be certified by the stamp of the supervisor and faxed to Central Command.
+
+station-goal-combat=
+    Dear station Command, due to the increase in attacks of pirate ships in this sector, the purpose of your shift is to raise the overall combat readiness of the station.
+
+    Required:
+    1. Organize an inspection of every sentient being and cargo arriving at or leaving the station.
+    2. Build or modify an existing security checkpoint at arrivals and departures. The checkpoint must be able to completely block the ports from the main part of the station.
+        Each of the above checkpoints must have at least one cell for the temporary detention of detainees.
+    3. Organize a spare weapons storage in the opposite part of the station from the brig.
+        The vault arsenal should have enough weapons and equipment to fully equip all security personnel.
+    4. Organize the recruitment of a new combat subdepartment of security.
+        Squad members must be recruited from the station's crew.
+        Recruitment should be carried out on a voluntary-compulsory basis.
+        Composition of the squad:
+            1 field medic;
+            1 field engineer;
+            3 combat operatives.
+        All members of the squad must be trained in all the necessary skills to conduct combat and fulfill their role.
+    5. Open a public shooting range.
+        The shooting range should present all available types of weapons or their training counterparts.
+        Avoid providing lethal weaponry to unauthorized personnel.
+    6. Encourage the use of the station boxing ring.
+        If there is no boxing ring, you must create one.
+
+station-goal-shuttle=
```

Comment:
Does this even make sense if Shipyard is ported?

Reply:
It's probably fine, because this would be more like making a custom shuttle. Shipyard system is buying shuttles from a shipyard, whereas this goal is turning your own station into a shipyard!